src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/FilterVariantTranchesIntegrationTest.java

```diff
@@ -0,0 +1,117 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.io.IOException;
+
+public class FilterVariantTranchesIntegrationTest  extends CommandLineProgramTest {
+
+
+    /**
+     * Run the tool on a small test VCF.
+     */
+    @Test
+    public void testTrancheFiltering() throws IOException {
+        final boolean newExpectations = false;
+
+        final String trancheVCF = largeFileTestDir + "VQSR/g94982_20_1m_10m_python_2dcnn.vcf.gz";
+        final String indelTruthVCF = largeFileTestDir + "VQSR/ALL.wgs.indels_mills_devine_hg19_leftAligned_collapsed_double_hit.sites.20.1M-10M.vcf";
+        final String snpTruthVCF = largeFileTestDir + "VQSR/Omni25_sites_1525_samples.b37.20.1M-10M.vcf";
+
+        final ArgumentsBuilder argsBuilder = new ArgumentsBuilder();
+        argsBuilder.addArgument(StandardArgumentDefinitions.VARIANT_LONG_NAME, trancheVCF)
+                .addArgument("resource", snpTruthVCF)
+                .addArgument("resource", indelTruthVCF)
+                .addArgument("tranche", "99.0")
+                .addArgument("info-key", "MIX_SMALL_2D_W_DROPOUT")
+                .addArgument(StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false");
+
+        if(newExpectations){
+            argsBuilder.addArgument(StandardArgumentDefinitions.OUTPUT_LONG_NAME, largeFileTestDir + "VQSR/expected/g94982_20_1m_10m_tranched_99.vcf");
+            runCommandLine(argsBuilder);
```

Comment:
This code path and the variable `newExpectations` can be removed, here and below.

Reply:
Done.